assume/common/forecasts.py

```diff
@@ -82,6 +84,21 @@ def get_price(self, fuel_type: str) -> pd.Series:
 
         return self[f"fuel_price_{fuel_type}"]
 
+    def _to_fast_series(self, value, name: str) -> FastSeries:
+        """
+        Converts a value to a FastSeries based on self.index.
+
+        Args:
+            value (float | list | pd.Series): The value to convert.
+            name (str): Name of the series.
+
+        Returns:
+            FastSeries: The converted FastSeries.
+        """
+        if isinstance(value, pd.Series):
+            value = value.values  # Use the values as an array for consistency
+        return FastSeries(index=self.index, value=value, name=name)
+
 
 class CsvForecaster(Forecaster):
```

Comment:
Could that be general? So from df to fastseries dictionary? Does not need to be related to forecaster.

Reply:
replaced with a method from FastSeries itself